python/py-rsa/Portfile

```diff
@@ -16,14 +16,12 @@ long_description    ${description}.\
     Python library as well as on the commandline.
 
 homepage            http://stuvel.eu/rsa
-master_sites        pypi:r/rsa
-distname            rsa-${version}
 
-checksums           rmd160  ed031a093373ab3c1a6d27d2d9e58235c0058809 \
-                    sha256  25df4e10c263fb88b5ace923dd84bf9aa7f5019687b5e55382ffcdb8bede9db5 \
-                    size    40956
+checksums           rmd160  399e2100a851fc00984b814ab614b6d5292a4f7a \
+                    sha256  109ea5a66744dd859bf16fe904b8d8b627adafb9408753161e766a92e7d681fa \
+                    size    46954
 
-python.versions     27 35 36 37 38
+python.versions     38
```

Comment:
This will break other ports that still depend on the pre-py38 subports, such as py-google-auth.

Reply:
OK, reversed